packages/core/src/utilities/convertVolumeToStackViewport.ts

```diff
@@ -0,0 +1,99 @@
+import * as Types from '../types';
+import cache, { ImageVolume } from '../cache';
+import { ViewportType } from '../enums';
+import uuidv4 from './uuidv4';
+
+/**
+ * Converts a volume viewport to a stack viewport.
+ *
+ * @param params - The parameters for the conversion.
+ * @param params.viewport - The volume viewport to convert.
+ * @param params.options - The conversion options.
+ * @param [params.options.viewportId] - The new stackViewportId, If not provided, the volume viewport id will be used.
+ * @param [params.options.background] - The background color of the stack viewport.
+ * @param [params.options.decache] - Whether to decache the volume. Defaults to false.
+ *
+ * @returns The converted stack viewport.
+ */
+async function convertVolumeToStackViewport({
+  viewport,
+  options,
+}: {
+  viewport: Types.IVolumeViewport;
+  options: {
+    viewportId?: string;
+    background?: Types.Point3;
+  };
+}): Promise<Types.IVolumeViewport> {
+  const volumeViewport = viewport;
+  const { id, element } = volumeViewport;
+  const renderingEngine = viewport.getRenderingEngine();
+  const imageIdIndex = viewport.getCurrentImageIdIndex();
+
+  const { background } = options || { volumeId: uuidv4() };
```

Comment:
Why do you generate a volumeId here and then throw it away?  Seems very odd.  You aren't even allowed to specify  volumeId in the options type.